### PR TITLE
chore: npm install warnings

### PR DESCRIPTION
**Pull request by medjedovicm**
## Intent
Resolved npm install warnings by updating `jest-cli` package.

## Checks

- [x] Code is formatted correctly (`npm run lint:fix`).
image
- [x] All unit tests are passing (`npm test`).
Mocked:
image
Server:
image
- [x] All CI checks are green.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,17 +88,16 @@
     "@types/xml": "1.0.6",
     "babel-jest": "27.2.4",
     "copyfiles": "2.4.1",
-    "ghooks": "2.0.4",
+    "husky": "^7.0.4",
```
Comment (allanbowe) [inline package.json:91]: What is the licence for this?

**Review by allanbowe — CHANGES_REQUESTED** [review]
I don't believe the latest version of husky is free for commercial use (but correct me if I'm wrong)

File: package.json (hunk 1 above)
Comment (allanbowe) [reply package.json:91]: MIT (now) it seems

**Comment by allanbowe** [comment]
I was wrong - they changed their licence in version 7:  https://github.com/typicode/husky/pull/916/files#diff-c693279643b8cd5d248172d9c22cb7cf4ed163a3c98c8a3f69c2717edd3eacb7

**Review by allanbowe — DISMISSED** [review]

**Review by saadjutt01 — APPROVED** [review]

**Review by VladislavParhomchik — APPROVED** [review]

**Pull request merged**